.buildbot.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# vim: expandtab sts=0 sw=4 smarttab
```

Comment:
I don't think we should include editor config in files, otherwise, since people use different editors, we'll end up with editor wars over whose comment go at the top ;)